### DISCUSSION TITLE
Support for touch devices (right-click functionality)

### DISCUSSION THREAD

**Discussion by DavidZweistein** [pimcore/pimcore#14159] [Ideas]
### Improvement description

Pimcore isnt usable with exclusively touch devices, since it depends strongly on right clicking actions, which devices such as the iPad dont support (without accessories). Some kind of mini-button menu at each element or something else would be great.

**Comment by mattamon** [comment]
Hey @DavidZweistein,

Thank you for your contribution! 
I am going to move this issue into the discussions, so we gather more feedback from the community.
We are currently finalizing Pimcore XI and do not see this feature to be part of the launch.
